tests/components/emoncms/test_sensor.py

```diff
@@ -59,12 +126,14 @@ async def test_coordinator_update(
         "success": True,
         "message": [get_feed(1, unit="°C")],
     }
-    await async_setup_component(hass, SENSOR_DOMAIN, emoncms_yaml_config)
-    await hass.async_block_till_done()
-    feeds = get_feeds([1])
-    for entity_id in get_entity_ids(feeds):
-        state = hass.states.get(entity_id)
-        assert state == snapshot(name=entity_id)
+    await setup_integration(hass, config_single_feed)
+
+    entity_entries = er.async_entries_for_config_entry(
+        entity_registry, config_single_feed.entry_id
+    )
+    for entity_entry in entity_entries:
+        state = hass.states.get(entity_entry.entity_id)
+        assert state == snapshot(name=entity_entry.entity_id)
```

Comment:
use the snapshot_platform helper for this at it will also capture the entity itself

Reply:
done